modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/CustomTooltip.js

```diff
@@ -11,19 +11,36 @@
 
 import PropTypes from 'prop-types';
 import React from 'react';
-
 /**
  * Component to customize the content of recharts Tooltip
  * http://recharts.org/en-US/api/Tooltip#content
  */
 export default function CustomTooltip(props) {
-	const {formatter, label, labelFormatter, payload, separator = ''} = props;
+	const {
```

Comment:
This component is weird. Published gets printed every time??

Reply:
The publish date has to be displayed only in the information about that day.